### DISCUSSION TITLE
Option to build with an existing conda

### DISCUSSION THREAD

**Discussion by dagewa** [dials/dials#1595] [Ideas]
@biochem-fan reports that building DIALS with an existing conda set up is possible by editing the `conda_exe` variable in [`bootstrap.py`](https://github.com/dials/dials/blob/0fabc4f7ba5c58f495a48783811441cbd87af6fd/installer/bootstrap.py#L99). Is this something we should enable as an option to `bootstrap.py`? It may help for integrating DIALS within other systems.

**Comment by Anthchirp** [comment]
fwiw, I deliberately removed that feature from our bootstrap, as it just makes support harder. In my opinion we should not support that - bootstrap should give you a well defined _development_ installation (specifically: for developing DIALS). All other use cases should install DIALS, ideally from a conda-forge package rather than bootstrap.

**Reply by dagewa** [reply]
That's reasonable. When we talk about a new _installer_, are we keeping in mind the situation where the user wants to use their own Python, in order to build DIALS into some larger system? In that case, what requirements would there be of that Python (such as, should it be a conda-aware Python)?
